### ISSUE TITLE
Are multiple GPUs available?

### ISSUE THREAD

**Issue by koreadhkim** [YapengTian/TDAN-VSR-CVPR-2020#36]
Are multiple GPUs available?

**Comment by YapengTian** [comment]
Yes, we can use multiple gpus during training (--gpu 0,1,3,...). For evaluation, we only use one gpu.

**Comment by sljlp** [comment]
how to evaluate an image with a large size, such as 1920x1080 or 1280x720?

**Comment by YapengTian** [comment]
One commonly used way is to split the full images into image patches and then compose the generate patches to full images again. https://github.com/YapengTian/TDAN-VSR-CVPR-2020/blob/master/solver.py pls look at the chop_forward function.
